CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Adds support to API300 HPE Synergy Image Streamer resources:
   - image_streamer_artifact_bundle
   - image_streamer_os_build_plan
 
+Enhancements:
+  - connection_template now supports `EthernetNetwork`, `FCoENetwork`, `FCNetwork` and `NetworkSet`
```

Comment:
Hey guys, I've completed the unit tests and will be pushing to this PR soon. Though, it's good to note for future reference that what we usually mean by 'New functionality includes testing.' is actually the inclusion of those unit tests referring to what you might have implemented or changed.